packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -54,14 +54,16 @@ import 'thumb_painter.dart';
 class CupertinoSwitch extends StatefulWidget {
   /// Creates an iOS-style switch.
   ///
```

Comment:
Cool, 2 small nit picks for this PR and in general.

1- Keep all documentations and comments in terse but full sentence forms (so with all the articles, punctuations etc).

```dart
/// The [value] parameter must not be null.
/// The [dragStartBehavior] parameter defaults to DragStartBehavior.start and must not be null 
```

2- Each class field's documentation, when extracted, will end up in its own web page (such as https://docs.flutter.io/flutter/cupertino/CupertinoSwitch/value.html) and needs to be able to stand alone. Document this in line 70 as well.